### DISCUSSION TITLE
Ender 3v2 with both Sprite Pro and BTT SKR Mini E3 V3

### DISCUSSION THREAD

**Discussion by Homer-S** [mriscoc/Ender3V2S1#632] [Q&A] [answered]
Awesome firmware!

So, I've found these two for my particular mix of upgrades

1) [Ender3V2-SKRME3V3-BLTUBL-MPC-20221222.bin](https://github.com/mriscoc/Ender3V2S1/releases/download/20221222/Ender3V2-SKRME3V3-BLTUBL-MPC-20221222.bin)
2) [Special Configurations](https://github.com/mriscoc/Special_Configurations/releases/tag/T13)

I can run 1 above for now missing out on high temperatures but, how do I get both the Sprite Pro and the SKR Mini playing together?

Homer

**Comment by Homer-S** [comment, marked as answer]
UPDATE,

I think I have figured it out but I have one open question.

I followed the [Professional Firmware Configurator](https://github.com/mriscoc/Special_Configurations), selected the list below, 

- Ender3V2
- SKRME3V3
- BLT w/UBL checked
- T13
- MPC

Then moved the Ender3V2-SKRM3V3-BLTUBL-T13-MPC configuration folder over, replaced the two config files and the platform.io file as directed and clicked build.

The Auto Build Marlin is asking which of these two I want to build. Not sure which one to use.

- STM32G0B1RE_btt
- STM32G0B1RE_btt_xfer

Updated update: looks like the first one is right based on this discussion: https://github.com/mriscoc/Ender3V2S1/discussions/131#discussioncomment-2860404

FAILURE, not sure where to go now. This is the terminal output.

[terminal01.txt](https://github.com/mriscoc/Ender3V2S1/files/10388191/terminal01.txt)

Homer

**Reply by mriscoc** [reply]
You need to download the configuration files that match the date of the main firmware repository release

https://github.com/mriscoc/Special_Configurations/archive/refs/heads/main.zip

**Reply by Homer-S** [reply]
That worked like a charm! Now just waiting on Amazon to deliver the BTT board to unrubble my printer...

Makes total sense in hindsight. How do you navigate to find the current one so when things rev, I can make sure to get the new Special_Configurations file? Nevermind... it's the one at the top, d'oh!

Update: Flashed on the board and was able to heat both the hot end and bed. Seems I need to do some wiring adjustments on the CR Touch and I should be fully operational. I'll print for a while before attempting either Linear Advance, Input Shaping or both.

Thanks!
Homer

**Comment by Homer-S** [comment]
To get my CR Touch working. I had to change one line.

#define Z_MIN_PROBE_PIN **PC14**

For some reason I also had to rewire the Sprite Extruder Pro cable to match the same order as the stock CR Touch cable.

Homer
